pdb2pqr/hydrogens/__init__.py

```diff
@@ -176,10 +176,12 @@ def switchstate(self, states, amb, state_id):
             # flag the added hydrogen
             residue.get_atom(hname).titratableH = True
             residue.get_atom(hname).addIntraBond(boundname)
+        return None
```

Comment:
https://docs.astral.sh/ruff/rules/implicit-return/

Reply:
That seems unnecessary.  Have other communities vetted the ruff rules?

Reply:
Not all ruff rules are necessary. It's by choice but it just brings cleaner code. If you don't like the rule we can just turn this off.

The same lint rule exists for flake8 and yes ruff is popular.

Reply:
The rule is now ignored and I reverted this.